r/R/csv.R

```diff
@@ -478,7 +478,7 @@ CsvWriteOptions$create <- function(include_header = TRUE, batch_size = 1024L) {
   )
 }
 
-readr_to_csv_read_options <- function(skip, col_names, col_types) {
+readr_to_csv_read_options <- function(skip = 0, col_names = TRUE, col_types = NULL) {
```

Comment:
Given it's not used here, should we just remove the `col_types` argument entirely?

Reply:
OK, I see we use it later when we're listing which ones we do and do not support.  Maybe this bit of the code needs refactoring in other ways, as it feels a bit brittle to depend on this here?